tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyForum.java

```diff
@@ -487,6 +487,7 @@ private void setupDB(Connection conn) throws SQLException {
             conn.commit();
         }
         reloadLocales(conn);
+        new SurveyForumCheck(conn).run();
```

Comment:
should this run in a separate thread if it's not a blocking operation? 

```suggestion
                SurveyThreadManager.getExecutorService().submit(() -> new SurveyForumCheck(conn).run());
```

Reply:
^ it should have its own Connection if it's a separate thread.

Reply:
Maybe. It's surprisingly fast ("0 sec.") and only happens once, during startup. I worry more about thread safety than performance here; still, I'm open to the idea...

Reply:
it'll be safe if it has its own Connection. and anyway, it may be "zero" seconds, but overall startup does take a little time.

Reply:
how's this: https://github.com/unicode-org/cldr/pull/2536